internal/clients/team3/clientgeneral.go

```diff
@@ -356,13 +338,3 @@ func (c *client) ResourceReport() shared.ResourcesReport {
 	skewedResource := resource / shared.Resources(c.params.resourcesSkew)
```

Comment:
can this ever be 0?

Reply:
I'm not sure, div 0 check has been added